src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int number = 100;
```

Comment:
what about access modifier? And format constant name https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names